proxy/src/main/java/com/velocitypowered/proxy/connection/backend/BungeeCordMessageResponder.java

```diff
@@ -172,7 +172,7 @@ private void processMessageRaw(ByteBufDataInput in) {
   }
 
   private void processMessage0(ByteBufDataInput in,
-      ComponentSerializer<Component, ?, String> serializer) {
+                               ComponentSerializer<Component, ?, String> serializer) {
```

Comment:
Lots of unnecessary reformatting. Please keep your PR focused on the feature at hand.